frontend/src/App.css

```diff
@@ -1,5 +1,24 @@
+:root {
+  --white-shade-0: white;
+  --white-shade-1: rgb(250, 250, 250);
+  --white-shade-2: rgb(219, 219, 219);
+}
+
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
 }
+
+.app-body {
+  background-color: var(--white-shade-1);
+}
+
+.flex-box {
+  display: flex;
+}
+
+* {
```

Comment:
The selector should be `body` instead of `*`. The difference is this:
- `*` will set the `font-size` properties on **EVERY** element in the DOM. This means all headings (`h1` to `h6`), `p` and `small` tags will have the same `font-size`, which is wrong
- `body` will only set this on the `body` element. However, due to inheritance, many other elements will inherit these styles.